src/java/picard/sam/MergeSamFiles.java

```diff
@@ -52,7 +52,17 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Merges multiple SAM/BAM files into one file.",
+        usage = "Merges multiple SAM/BAM files into one file." +
+                "<br />" +
+                "<h4>Usage example:</h4>" +
+                "<pre>" +
+                "java -jar picard.jar MergeSamFiles \\<br />" +
+                "     -I=MyBAM1.bam \\<br />" +
+                "     -I=MyBAM2.bam \\<br />" +
+                "     -O=/Users/MymergedBAM.bam" +
```

Comment:
Leave out "/Users/", users might think that that part of the path is meaningful.